srcpkgs/python3.7/template

```diff
@@ -0,0 +1,102 @@
+#
+pkgname=python3.7
+version=3.7.4
+revision=1
+wrksrc="Python-${version}"
+short_desc="Interpreted, object-oriented programming language (${version%.*})"
+maintainer="Cameron Nemo <cnemo@tutanota.com>"
+license="Python-2.0"
+homepage="https://www.python.org"
+distfiles="https://www.python.org/ftp/python/${version}/Python-${version}.tar.xz"
+checksum=fb799134b868199930b75f26678f18932214042639cd52b16da7fd134cd9b13f
+
+pycompile_dirs="usr/lib/python${version%.*}"
+hostmakedepends="pkg-config"
+makedepends="libffi-devel readline-devel gdbm-devel libressl-devel expat-devel
+ sqlite-devel bzip2-devel zlib-devel liblzma-devel"
+depends="ca-certificates"
+
+alternatives="
+ python:idle:/usr/bin/idle${version%.*}
+ python:2to3:/usr/bin/2to3-${version%.*}
+ python:pydoc:/usr/bin/pydoc${version%.*}
+ python:python:/usr/bin/python${version%.*}
+ python:python.1:/usr/share/man/man1/python${version%.*}.1"
+
+pre_configure() {
+	# Ensure that internal copies of zlib, expat and libffi are not used
+	rm -r Modules/expat
+	rm -r Modules/_ctypes/{darwin,libffi}*
+}
+do_configure() {
+	local _args
+	unset GCC CC CXX CPP LD AR AS RANLIB
+	if [ "$CROSS_BUILD" ]; then
+		mkdir -p host-build
+		cd host-build
+		env -i PATH=$PATH CFLAGS=-Os ../configure
+		env -i PATH=$PATH make ${makejobs} python
+		mkdir -p ../hostpython
+		mv python ../hostpython
+		cd ..
+		patch -Np0 -i ${FILESDIR}/cross.patch
+		_args="--build=${XBPS_MACHINE%%-musl}"
+	fi
+	./configure ${configure_args} ${_args} \
+		--enable-shared --enable-ipv6 --enable-loadable-sqlite-extensions \
+		--with-threads --with-computed-gotos --with-dbmliborder=gdbm:ndbm \
+		--with-system-expat --with-system-ffi --without-ensurepip
+}
+do_build() {
+	export PATH="$PATH:$wrksrc/hostpython"
+	make ${makejobs}
+}
+do_install() {
+	export PATH="$PATH:$wrksrc/hostpython"
+	make DESTDIR=${DESTDIR} install maninstall
+	vlicense LICENSE
+
+	rm -f ${DESTDIR}/usr/bin/2to3
+	# Remove files that belong to python3-tkinter and idle-python3
+	rm -f ${DESTDIR}/usr/bin/idle3*
+	rm -rf ${DESTDIR}/usr/lib/python${version%.*}/idlelib
+	rm -rf ${DESTDIR}/usr/lib/python${version%.*}/tkinter
+	rm -rf ${DESTDIR}/usr/lib/python${version%.*}/turtledemo
+	rm -f ${DESTDIR}/usr/lib/python${version%.*}/turtle.py
+	# Remove test module and tests that fail to be byte-compiled.
+	rm -rf ${DESTDIR}/usr/lib/python${version%.*}/test
+	rm -rf ${DESTDIR}/usr/lib/python${version%.*}/lib2to3/tests
+	# Remove references to the install(1) wrapper.
+	sed -e "s,${XBPS_WRAPPERDIR},/usr/bin,g" -i \
+		${DESTDIR}/usr/lib/python${version%.*}/_sysconfigdata_m_*.py \
+		${DESTDIR}/usr/lib/python${version%.*}/config-${version%.*}m*/Makefile
+	if [ "$CROSS_BUILD" ]; then
+		# Remove references to cross toolchain.
+		sed -i "s/$XBPS_CROSS_TRIPLET-//g" \
+			${DESTDIR}/usr/lib/python${version%.*}/_sysconfigdata_m_*.py \
+			${DESTDIR}/usr/lib/python${version%.*}/config-${version%.*}m*/Makefile
+	fi
+}
+
+post_install() {
+	rm -f -- "${DESTDIR}/usr/lib/libpython3.so"
```

Comment:
this lib does not seem to be used by anyone or anything: `for pkg in $(xbps-query -RX python3); do xbps-query -RS "$pkg" | grep libpython3.so; done`